pkg/operator/ceph/cluster/watcher_test.go

```diff
@@ -219,6 +219,10 @@ func TestHandleNodeFailure(t *testing.T) {
 				CSI: &corev1.CSIPersistentVolumeSource{
 					Driver:       fmt.Sprintf("%s.rbd.csi.ceph.com", ns),
 					VolumeHandle: "0001-0009-rook-ceph-0000000000000002-24862838-240d-4215-9183-abfc0e9e4002",
+					VolumeAttributes: map[string]string{
+						"pool":      "replicapool",
+						"imageName": "csi-vol-58469d41-f6c0-4720-b23a-0a0826b841ca",
+					},
```

Comment:
Add a test for static PVC and a PVC created from another csi driver (which is not deployed by Rook) as well?

Reply:
added test